pom.xml

```diff
@@ -412,12 +412,12 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-collections4</artifactId>
-        <version>4.4</version>
+        <version>4.5.0-M3</version>
```

Comment:
`-M3` indicates a prerelease milestone version, not a final version, and should be avoided. The last final version should be used instead, which I think is still 4.4